src/pthread_private.h

```diff
@@ -48,17 +48,18 @@ struct pthread_t_
 struct pthread_attr_t_ {
     uint32_t 	dwStackSize ;
     uint32_t 	dwCreatingFlag ;
-    int 			priority ;
-    int         	detachstate;
+    int 	priority ;
+    int         detachstate;
 };
 
 struct pthread_condattr_t_ {
-	int 			dummy;
+    int         clock; // if 0 == we use real_clock; if == 1 we use monotonic_clock

```

Comment:
Why not use `clockid_t clock_id` directly here which will remove this 0->CLOCK_REALTIME and 1->CLOCK_MONOTONIC mapping?